_episodes/01-intro.md

```diff
@@ -157,7 +157,7 @@ weight in kilograms is now: 65.0
 > ~~~
 > {: .output}
 >
-> ![Creating Another Variable](../fig/python-sticky-note-variables-02.svg)
+> ![Creating additional variable example: old variable weight_kg equals 65.0 and new variable weight_lb equals 143.0](../fig/python-sticky-note-variables-02.svg)
```

Comment:
I think we could lose some of the starting text to make this shorter. What about something like `value of 65.0 with weight_kg name stuck on it and value of 143.0 with weight_lb name stuck on it`?